inc/class-sitemaps.php

```diff
@@ -117,11 +117,15 @@ function __construct() {
 	 * Check the current request URI, if we can determine it's probably an XML sitemap, kill loading the widgets
 	 */
 	public function reduce_query_load() {
-		if ( isset( $_SERVER['REQUEST_URI'] ) && ( in_array( substr( $_SERVER['REQUEST_URI'], - 4 ), array(
-				'.xml',
-				'.xsl',
-			) ) )
-		) {
+
+		if ( ! isset( $_SERVER['REQUEST_URI'] ) ) {
+			return;
+		}
+
+		$request_uri = $_SERVER['REQUEST_URI'];
```

Comment:
The condition had high enough WTF/minute for me to overcome laziness and simplify, simple as that. :)